package.json

```diff
@@ -26,7 +26,8 @@
     "gen:network": "ts-node ./scripts/genNetwork.ts",
     "prepublishOnly": "yarn build && yarn format",
     "preversion": "yarn lint",
-    "build": "./scripts/builder",
+    "prebuild": "yarn gen:abi",
```

Comment:
make sure to always generate abis when running build (and build is run on prepublish)